### PR TITLE
1655-add-ontocompany-ontology: Added .csv and .owl files for the Onto…

### PR DESCRIPTION
**Pull request by nagarajankarthik**
…Company ontology.

### CODE REVIEW

File: JPS_Ontology/ontology/ontochemplant/ontochemplant.csv
Comment (msff2) [inline JPS_Ontology/ontology/ontochemplant/ontochemplant.csv:1]: Replace "http://www.theworldavatar.com/kg/ontocompany/OntoCompany.owl" with "http://www.theworldavatar.com/kg/ontocompany/" on row 2.

OntoCompany is an ontology defined for representing the processes, business and geospatial information of a chemical plant. The definition of the ontology does not seem to correctly describe what this ontology is as it emphasises the chemical plant, so rewrite "An ontology defined for representing the processes, business and geospatial information of a chemical plant" on row 4.

Do not provide complete IRIs for the classes that are created in this ontology. Therefore, replace "http://www.theworldavatar.com/kg/ontocompany/Factory" with "Factory" and "http://www.theworldavatar.com/kg/ontocompany/Industry" with "Industry".

Replace "An industrial plant" with "A factory" in the following definitions of classes:

"An industrial plant established to produce printed products."
"An industrial plant established to produce precision engineering products."
"An industrial plant established to produce food."
"An industrial plant established to produce beverages."
"An industrial plant established to produce pharmaceuticals."
"An industrial plant established to produce chemicals."

Modify the definition of DataCentre on row 14 as "A centre for housing servers and storage devices that manage and store data.".

Modify the definition of Industry on row 15 as "An economic sector that groups businesses that offer products or services.".

Classes that are reused from other ontologies should be provided with their complete IRIs, so fix the following classes:
- ChemicalPlant
- ChemicalIndustry
- GeneratedHeat

In the input for hasLotArea in rows 42 and 43, put "exactly 1" in row 42 and delete 43. The current TBoxGeneration can produce the intended result using a single input line with the quantifier. Similarly, reduce rows 53 and 54 that describe hasSpecificEnergyConsumption to a single row. Do the same for https://www.theworldavatar.com/kg/ontobuiltenv/hasOntoCityGMLRepresentation in rows 44 and 45.

You need to include the belongsToIndustry object property in this file to establish the relationship between plants and their industries.

**Review by msff2 — CHANGES_REQUESTED** [review]

File: JPS_Ontology/ontology/ontochemplant/ontochemplant.csv
Comment (nagarajankarthik) [reply JPS_Ontology/ontology/ontochemplant/ontochemplant.csv:1]: I have made these changes in commit e0106702c7f4329bb7cc545a9b0f3205deb2b163. The GeneratedHeat class is transferred to ontocompany. Similarly, ChemicalIndustry is defined in OntoCompany and not OntoChemPlant. I have updated ontochemplant.csv accordingly.

**Review by msff2 — APPROVED** [review]
I have refined the CSV file, generated the corresponding OWL file and edited the OWL file for the belongs-to relationship between each plant and its related industry. This manual work is necessary because the TBox Generator does not support creating these subclass of axioms.

**Comment by sm453** [comment]
> edited the OWL file for the belongs-to relationship between each plant and its related industry. This manual work is necessary because the TBox Generator does not support creating these subclass of axioms.

@msff2, can you please add a short `readme.md` file that explains 1) the fact that you edited the OWL file (post-generation), and 2) why.

**Comment by msff2** [comment]
> > edited the OWL file for the belongs-to relationship between each plant and its related industry. This manual work is necessary because the TBox Generator does not support creating these subclass of axioms.
> 
> @msff2, can you please add a short `readme.md` file that explains 1) the fact that you edited the OWL file (post-generation), and 2) why.

I have added a short `readme.md` file briefly explaining the reason for the edit.

**Pull request merged**